packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-node-ciemss.vue

```diff
@@ -192,69 +189,42 @@ const isChartsEmpty = computed(
 	() => _.isEmpty(interventionCharts.value) && _.isEmpty(variableCharts.value) && _.isEmpty(comparisonCharts.value)
 );
 
-const isFinished = (state: ProgressState) =>
-	[ProgressState.Cancelled, ProgressState.Failed, ProgressState.Complete].includes(state);
-
-// Handle simulation status update event for the forecast run
-useClientEvent(
-	ClientEventType.SimulationNotification,
-	async (event: ClientEvent<StatusUpdate<SimulationNotificationData>>) => {
-		const simulationNotificationData = event.data.data;
-		if (simulationNotificationData.simulationId !== inProgressForecastId.value || !isFinished(event.data.state)) return;
-
-		const simId = simulationNotificationData.simulationId;
-		let errorMessage = { name: '', value: '', traceback: '' };
-		let forecastId = '';
-		if (event.data.state === ProgressState.Failed) {
-			const simulation = await getSimulation(simId);
-			if (simulation?.status && simulation?.statusMessage) {
-				errorMessage = {
-					name: simId,
-					value: simulation.status,
-					traceback: simulation.statusMessage
-				};
-			}
-		} else if (event.data.state === ProgressState.Complete) {
-			forecastId = simId;
-		}
-		const state = _.cloneDeep(props.node.state);
-		state.inProgressForecastId = '';
-		state.forecastId = forecastId;
-		state.errorMessage = errorMessage;
-		emit('update-state', state);
-		if (event.data.state === ProgressState.Complete) await processResult(simId);
+const poller = new Poller();
+const pollResult = async (runId: string) => {
+	poller.setPollAction(async () => pollAction(runId));
+	const pollerResults = await poller.start();
+	const state = _.cloneDeep(props.node.state);
+	state.errorMessage = { name: '', value: '', traceback: '' };
+
+	if (pollerResults.state === PollerState.Cancelled) {
+		// state.inProgressForecastId = '';
```

Comment:
Confusing to leave these in here as commented out

Reply:
No reason, that's a typo